assignments/objects.js

```diff
@@ -49,16 +86,38 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  speak: function(){
+    console.log(`Hi I'm ${this.name}`);
+  },
+  child: {
+    name: "George",
+    age: 50,
+    speak: function(){
+      console.log(`Hi I'm ${this.name}`);
+    },
+    grandchild: {
+      name: "Sam",
+      age: 30,
+      speak: function(){
+        console.log(`Hi I'm ${this.name}`);
+      }
+    }
+  }
+}
 
 // Log the parent object's name
-
+console.log(parent.name);
 // Log the child's age
-
+console.log(parent.child.age);
 // Log the name and age of the grandchild
+console.log(parent.child.grandchild.name, parent.child.grandchild.age);
 
 // Have the parent speak
-
+parent.speak();
 // Have the child speak
-
+parent.child.speak();
 // Have the grandchild speak
+parent.child.grandchild.speak();
```

Comment:
This is great! We didn't have this when I went through, when I saw it yesterday I felt a little faint. Looks like you breezed through!